### PR TITLE
Resumy App

### PR DESCRIPTION
**Pull request by lndgalante**
## How to use it ?

1. Download this <a href="https://jsonresume.org/schema/">schema</a> into a JSON file

2. Go to the section "My Resumes"

3. Drag and Drop the JSON file

4. Display it!

## Features

- [x] Drag and Drop

- [x] Resume builder (WIP)

## Further ideas and features
- Themes
- Code refactoring
- Improved resume layout
- Finish Resume builder (Create Resume section)

Link: [Resumy](https://resumy.now.sh/)

Screenshot of final push:
Captura de Pantalla 2020-03-28 a la(s) 3 56 11 p  m

### CODE REVIEW

**Comment by saralaffin** [comment]
Sorry I'm having a hard time finding what the resume looks like... but awesome idea for an app!

File: resumy/src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <Dashboard />
+      </Route>
+      <Route path='/my-resumes' exact>
+        <MyResumes />
+      </Route>
+      <Route path='/create-resume' exact>
+        <CreateResume />
+      </Route>
+    </Switch>
+  </Router>,
```
Comment (tammiec) [inline resumy/src/index.js:24]: Great use of react-router to implement extra features!

**Review by tammiec — COMMENTED** [review]
Really clean code, and you accomplished a LOT in the little time we had! Would love to see a deployed version to see what the UI actually looks like - quite hard to visualize otherwise!

**Comment by lndgalante** [comment]
Thanks @saralaffin @tammiec for your comments the link was in the PR description:

https://resumy.now.sh/

**Comment by Paahn** [comment]
> Really clean code, and you accomplished a LOT in the little time we had! Would love to see a deployed version to see what the UI actually looks like - quite hard to visualize otherwise!

actually is deployed, find link above or here https://resumy.now.sh/my-resumes

**Comment by Paahn** [comment]
really nice work, especially enjoyed the drag and drop functionality! my only constructive criticism from a developer's perspective would be to commit to your project more frequently, ie have more than 2 commits.

**Comment by lndgalante** [comment]
Yes that's a great comment @Paahn thanks for that suggestion.
I was so nervous that I almost to commit at all haha.

**Pull request closed without merging**